.github/workflows/test.yaml

```diff
@@ -55,6 +55,9 @@ jobs:
                 os: [ubuntu-latest, macos-latest]   # eventually add `windows-latest`
                 python-version: [3.7, 3.8, 3.9, "3.10"]
 
+        env:
+          GITHUB_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

Comment:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
```suggestion
          GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```